tests/sap/aibus/dar/client/test_data_manager_client.py

```diff
@@ -456,6 +456,9 @@ def test_wait_for_validation_raises_if_validation_fails(self):
 
             dataset_response = self._make_dataset_response(bad_dataset_state)
 
+            if bad_dataset_state == "VALIDATION_FAILED":
+                dataset_response["validationMessage"] = "Detailed Description"
```

Comment:
Would you want to keep the validation message as `Detailed Description`? Is this on purpose?

Reply:
This is just for the sake of the test to show that the exact value from the server is passed through to the user. I might as well have used `Foobar` :)

Reply:
Oh yes, I missed out on the fact that this was a test. Mea culpa :)